Produlytics/ui/back-end/src/main/java/it/deltax/produlytics/uibackend/common/detection/business/domain/DetectionLight.java

```diff
@@ -0,0 +1,8 @@
+package it.deltax.produlytics.uibackend.common.detection.business.domain;
+
+public record DetectionLight(
```

Comment:
So che dovrebbe essere solo un refactor, ma secondo me sarebbe da cambiare questo nome. In particolare visto che le entità ora si chiamano `DetectionEntity` questa si può chiamare `Detection` e basta.

Reply:
Alberto e Leila hanno detto che ci penseranno loro

Reply:
Sì, è per questo che secondo me non ha senso copia-incollare dal PoC così come sono e piuttosto copia-incollarle quando le devi riprendere e cambiare.